gramps/gen/config.py

```diff
@@ -325,6 +325,67 @@ def emit(key):
 register('utf8.cremated-symbol', "⚱")
 register('utf8.killed-symbol', "x")
 
+register('spacing.lst_person', [])
```

Comment:
If we register these in the `__init__` method of the `PersistentTreeView` then it would make using this class easier.

Reply:
OK.